locale/pt-br/texts.po

```diff
@@ -0,0 +1,734 @@
+msgstr "Para importar sua wallet,"
+
+#: src/screens/InitWallet.js:300
+#, javascript-format
+msgid ""
+"You need to **write down the ${ this.numberOfWords } seed words** of your "
+"wallet, separated by space."
+msgstr ""
+"Você precisa **digitar as ${ this.numberOfWords } palavras** da sua wallet, "
+"separadas por espaço."
+
+#: src/screens/InitWallet.js:303
+msgid "Words"
+msgstr "Palavras"
+
+#: src/screens/InitWallet.js:308
+msgid "Enter your seed words separated by space"
+msgstr "Digite suas palavras separadas por espaços"
+
+#: src/screens/LoadHistoryScreen.js:143
+msgid "Loading your transactions"
+msgstr "Carregando suas transações"
+
+#: src/screens/LoadHistoryScreen.js:146
+#, javascript-format
+msgid "**${ _this3.state.transactions } transactions** found"
+msgstr "**${ _this3.state.transactions } transações** encontradas"
+
+#: src/screens/LoadHistoryScreen.js:149
+msgid "**${ _this3.state.addresses } addresses** found"
+msgstr "**${ _this3.state.addresses } endereços** encontrados"
+
+#: src/screens/MainScreen.js:95
+msgid "No transactions"
+msgstr "Nenhuma transação"
+
+#: src/screens/MainScreen.js:101
+msgid "Share your address"
+msgstr "Compartilhe seu endereço"
+
+#: src/screens/MainScreen.js:104
+msgid "with friends and start exchanging tokens"
+msgstr "com seus amigos e comece a trocar seus tokens"
+
+#: src/screens/MainScreen.js:444 src/screens/MainScreen.js:476
+msgid "Available Balance"
+msgstr "Saldo Disponível"
+
+#: src/screens/MainScreen.js:453
+msgid "Locked"
+msgstr "Bloqueado"
+
+#: src/screens/PaymentRequestDetail.js:69
+#, javascript-format
+msgid "You've just received **${ amount } ${ symbol }**"
+msgstr "Você recebeu **${ amount } ${ symbol }**"
+
+#: src/screens/PaymentRequestDetail.js:86
+msgid "PAYMENT REQUEST"
+msgstr "REQUISIÇÃO DE PAGAMENTO"
+
+#: src/components/TxDetailsModal.js:50 src/screens/PaymentRequestDetail.js:104
+msgid "Token"
+msgstr "Token"
+
+#: src/screens/PaymentRequestDetail.js:116 src/screens/Receive.js:29
+msgid "My Address"
+msgstr "Meu endereço"
+
+#: src/screens/PaymentRequestDetail.js:120
+msgid "Status"
+msgstr "Status"
+
+#: src/screens/PaymentRequestDetail.js:121
+msgid "Received"
+msgstr "Recebido"
+
+#: src/screens/PaymentRequestDetail.js:121
+msgid "Waiting confirmation"
+msgstr "Aguardando confirmação"
+
+#: src/screens/PinScreen.js:157
+msgid "Incorrect PIN Code. Try again."
+msgstr "PIN incorreto. Tente novamente."
+
+#: src/screens/PinScreen.js:53
+msgid "Enter your PIN Code "
+msgstr "Digite seu PIN"
+
+#: src/screens/PinScreen.js:54
+msgid "Unlock Hathor Wallet"
+msgstr "Desbloqueie sua Hathor Wallet"
+
+#: src/screens/PinScreen.js:169
+msgid "Cancel"
+msgstr "Cancelar"
+
+#: src/screens/PinScreen.js:172 src/screens/Settings.js:128
+msgid "Reset wallet"
+msgstr "Resetar a wallet"
+
+#. *
+#.      * index {number} Selected index of the tab bar
+#.      
+#.  eslint thinks routes is not used, but TabView uses it
+#.  eslint-disable-next-line react/no-unused-state
```

Comment:
What are these comments above?

Reply:
I'm sure why the `npm run locale-update-pot` decided to get these comments from the source-code.

Reply:
Ah ok, this is auto generated. Weird.